packages/blocks/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Enforce `blockType.parent` to be an array. While `parent` being a string wasn't never supported, it worked with some unintended side-effects that have been fixed by [#66250](https://github.com/WordPress/gutenberg/pull/66250). For that reason, we've added some code that automatically migrates strings to arrays — though it still raises a warning.
```

Comment:
Added a changelog to make sure this doesn't go unnoticed.

Reply:
We should also add a heading to the change log entry, but I'm not sure what that should be.. Maybe `code quality`? 🤔

Reply:
Added `Bug fixes` heading https://github.com/WordPress/gutenberg/pull/66250/commits/0c4a9f55ec3e9f081274255691b27a79757c939d